R/param_engine_xgboost.R

```diff
@@ -8,7 +8,15 @@
 #' @details
 #' For more information, see the [xgboost webpage](https://xgboost.readthedocs.io/en/latest/parameter.html).
 #' @examples
-#' scale_pos_weight()
+#'
```

Comment:
Could you change this example to a call to the parameter functions rather than the tuning example? I think it's generally a good example to have but is better placed in an article on how to tune rather than here in the infrastructure package. That should also make the tests pass again which are currently failing because tidymodels is not a declared dependency.

Reply:
Yes, will do!